### PR TITLE
Fix rpc-tester based on unsafe debug namespace

### PR DESCRIPTION
**Pull request by JayChoi1736**
this PR split debug namespace tester to debug, unsafedebug tester.

and script for [klaytn](https://github.com/klaytn/klaytn) CI also changed

### CODE REVIEW

File: debug/debug_rpc.py [hunk 1]
```diff
@@ -75,18 +75,6 @@ def test_debug_startPProf_success(self):
         method = f"{self.ns}_stopPProf"
         Utils.call_ws(self.endpoint, method, [], self.log_path)
```
Comment (2dvorak) [inline debug/debug_rpc.py:77]: Above lines, L21-L76, should be deleted, too.

Comment (JayChoi1736) [reply debug/debug_rpc.py]: resolve by [c319228](https://github.com/klaytn/klaytn-rpc-tester/pull/13/commits/c319228db76514cca92ba06e0837c6ce25c152b2)

**Comment by 2dvorak** [comment]
We should modify `create_params_for_starting_rpc` to include `unsafedebug` in `apis`.
Or using `namespace` in `config.json` could be better..

File: main.py [hunk 2]
```diff
@@ -715,7 +723,7 @@ def load_test_suites():
     rpc_test_suites = list()
     ws_test_suites = list()
 
-    namespaces = config.get("namespaces", "admin,debug,eth,governance,klay,net,personal,txpool")
+    namespaces = config.get("namespaces", "admin,debug,unsafe_debug,eth,governance,klay,net,personal,txpool")
```
[the remark this answers is not in the document]
Comment (JayChoi1736) [reply main.py]: resolve by [c6aff53](https://github.com/klaytn/klaytn-rpc-tester/pull/13/commits/c6aff536e65a1c25d2e3ce85cc0a58f4511be5e9)

[the remark this answers is not in the document]
Comment (2dvorak) [reply main.py]: Thank you!

**Review by 2dvorak — APPROVED** [review]
LGTM, thank you!

**Pull request merged**